widget/customization.md

```diff
@@ -0,0 +1,238 @@
+  - **outlined**: has thin borders as the default style.
+  - **elevation**: adds a shadow effect.
+  - **filled**: features a solid background color (based on the palette.background.paper property).
+- **MuiIconButton** handles icon-based buttons.
+- **MuiInputCard** is used for input forms.
+
+With the components option, you can customize each element's appearance using MUI’s `styleOverrides` feature, allowing precise styling adjustments.
+
+In this example, the `MuiButton` component has been modified to adjust the background color, text color, rounded border, and hover behavior of the button.
+
+```javascript
+import { BandoWidget, WidgetConfig } from "@bando/widget";
+import { useMemo } from "react";
+
+export const WidgetPage = () => {
+  const widgetConfig: WidgetConfig = useMemo(
+    () => ({
+      palette: {
+        primary: {
+          main: "#006Eff",
+        },
+        secondary: {
+          main: "#FFC800",
+        },
+        background: {
+          default: "#ffffff",
+          paper: "#f8f8fa",
+        },
+        text: {
+          primary: "#00070F",
+          secondary: "#6A7481",
+        },
+        grey: {
+          200: "#EEEFF2",
+          300: "#D5DAE1",
+          700: "#555B62",
+          800: "#373F48",
+        },
+      },
+      shape: {
+        borderRadius: 12,
+        borderRadiusSecondary: 12,
+        borderRadiusTertiary: 24,
+      },
+      container: {
+        boxShadow: "0px 8px 32px rgba(0, 0, 0, 0.08)",
+        borderRadius: "16px",
+      },
+      components: {
+        MuiCard: {
+          defaultProps: { variant: "filled" },
+        },
+        MuiButton: {
+          styleOverrides: {
+            root: {
+              backgroundColor: "#006Eff",
+              color: "#ffffff",
+              borderRadius: "8px",
+              "&:hover": {
+                backgroundColor: "#0051CC",
+              },
+            },
+          },
+        },
+      },
+    }),
+    []
+  );
+
+  return (
+    <BandoWidget integrator="Your dApp/company name" config={widgetConfig} />
+  );
+};
+```
+
+<!-- TODO: Show result -->
+
+<!-- TODO: Appearance in case is required -->
+
+### Disable Elements
+
+The `disabledUI` property lets you choose which parts of the widget’s interface should be disabled. This can help you restrict user interaction with specific sections of the widget that may not be needed or suitable for your implementation.
+
+The `DisabledUI` enum defines the UI elements that can be turned off to stop user interaction.
+
+```javascript
+export enum DisabledUI {
+  // Disables the input field for the token amount
+  FromAmount = 'fromAmount',
+  // Disables the button for the source token selection
+  FromToken = 'fromToken',
+  // Disables the button for specifying the destination address
+  ToAddress = 'toAddress',
+  // Disables the button for the destination token selection
+  ToToken = 'toToken',
+}
+```
```

Comment:
This is more Lifi. They're awesome as reference but lets adapt this to our spend use case. Like a ProductID and such.